src/features/myAccounts/index.tsx

```diff
@@ -191,7 +199,7 @@ const AccountsList = ({ safes, onLinkClick, isSidebar = false }: AccountsListPro
                 </Box>
 
                 {/* All Accounts */}
-                <Accordion sx={{ border: 'none' }}>
+                <Accordion sx={{ border: 'none' }} defaultExpanded={!isSidebar}>
```

Comment:
In the sidebar it is now always collapsed even if there are no pinned safes. Didn't we want to expand it if there are no pinned safes and keep it collapsed if there is at least one pinned safe?

Reply:
The defaultExpanded prop wasn't working because it only applies to the first render, and the safes haven't loaded yet. I'll look into it more tomorrow on the flight